### PR TITLE
Fix highlight reference

### PR DESCRIPTION
**Pull request by yann300**

### CODE REVIEW

File: src/app/editor/contextualListener.js [hunk 1]
```diff
@@ -115,10 +116,16 @@ class ContextualListener {
     let lastCompilationResult = this._deps.compilersArtefacts['__last']
     if (lastCompilationResult && lastCompilationResult.languageversion.indexOf('soljson') === 0) {
       let lineColumn = this._deps.offsetToLineColumnConverter.offsetToLineColumn(position, position.file, lastCompilationResult.getSourceCode().sources, lastCompilationResult.getAsts())
-      let css = 'highlightreference'
+      const css = csjs`
+        .highlightref_fullLine {
+          position:absolute;
+          z-index:20;
```
Comment (GrandSchtroumpf) [inline src/app/editor/contextualListener.js]: You should use a "real" value like "1" or "2" in some rare cases.

File: src/app/editor/contextualListener.js [hunk 2]
```diff
@@ -115,10 +116,16 @@ class ContextualListener {
     let lastCompilationResult = this._deps.compilersArtefacts['__last']
     if (lastCompilationResult && lastCompilationResult.languageversion.indexOf('soljson') === 0) {
       let lineColumn = this._deps.offsetToLineColumnConverter.offsetToLineColumn(position, position.file, lastCompilationResult.getSourceCode().sources, lastCompilationResult.getAsts())
-      let css = 'highlightreference'
+      const css = csjs`
+        .highlightref_fullLine {
+          position:absolute;
```
Comment (GrandSchtroumpf) [inline src/app/editor/contextualListener.js]: When using absolute, most of the time you want to specify the top / left. At least to make it explicit.

File: src/app/editor/contextualListener.js [hunk 3]
```diff
@@ -132,7 +139,7 @@ class ContextualListener {
       }
       const fileName = lastCompilationResult.getSourceName(position.file)
       if (fileName) {
-        return this.editor.addMarker(lineColumn, fileName, css)
+        return this.editor.addMarker(lineColumn, fileName, css.highlightref_fullLine.className)
```
Comment (GrandSchtroumpf) [inline src/app/editor/contextualListener.js]: Why not just writing "highlightref_fullLine" ?

File: src/app/editor/contextualListener.js (hunk 2 above)
Comment (yann300) [reply src/app/editor/contextualListener.js]: did the changes. except this one, anyway ace editor is handling these properties

**Pull request merged**